docs/json_schemas/dns_record/v0/provider.json

```diff
@@ -39,7 +39,7 @@
           "description": "Status for the domain request.",
           "examples": [
             "approved",
-            "invalid_credentials"
+            "permission_denied"
```

Comment:
If there's no auth, then what would trigger "permission denied"?

Reply:
The DNS policy charm is going to take on DNS record requests and allow an operator to approve them one by one. So I thought that it could be handy to leave this possible status to convey a disapproval from the operator to the requesting charm.